### PR TITLE
Add tabs for switching between stats and timetable on attendance page

### PR DESCRIPTION
**Pull request by banocean**
[Screencast from 2024-08-29 18-56-30.webm](https://github.com/user-attachments/assets/61af15d1-3f00-4dd0-8a79-699d1726371f)

### CODE REVIEW

File: patches/attendance/styles.css [hunk 1]
```diff
@@ -0,0 +1,29 @@
+.attendance-tabs > button {
+    padding-inline: 15px;
+    padding-block: 10px;
+    font-size: 15px;
+    background: var(--kolor-wcag);
```
Comment (PanLiszka) [inline patches/attendance/styles.css:5]: opera_gZAIWUxHP2
opera_zRNMz6NcKQ


Buttons doesn't look good with this background on eduVULCAN, maybe change color of font?

File: patches/attendance/tabs.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
+const selector = document.createElement("div")
+selector.innerHTML = "<button disabled>Ferekwencja</button><button>Statystyki</button>"
```
Comment (PanLiszka) [inline patches/attendance/tabs.js]: Ferekwencja

**Review by PanLiszka — CHANGES_REQUESTED** [review]

File: patches/attendance/styles.css (hunk 1 above)
Comment (banocean) [reply patches/attendance/styles.css:5]: imo it looks good enough

Comment (PanLiszka) [reply patches/attendance/styles.css:5]: ok

File: patches/attendance/styles.css [hunk 3]
```diff
@@ -6,6 +6,7 @@
     border: 0;
     cursor: pointer;
     border-radius: 25px;
+    font-family: "WorkSans", Graphik, -apple-system, BlinkMacSystemFont, "Segoe UI", Helvetica, Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol";
```
Comment (PanLiszka) [inline patches/attendance/styles.css:9]: so much better

**Review by PanLiszka — APPROVED** [review]

File: patches/attendance/styles.css (hunk 3 above)
Comment (banocean) [reply patches/attendance/styles.css:9]: I just used same fonts as them for everything else, so it's more blending in with the rest rn

**Pull request merged**